editions/tw5.com/tiddlers/customising/Alternative page layouts.tid

```diff
@@ -8,8 +8,39 @@ type: text/vnd.tiddlywiki
 
 ! Creating an alternative page layout
 
-Creating an alternative layout goes beyond [[adding or removing features|Customising TiddlyWiki's user interface]] from the default interface and allows you to create an entirely new layout from scratch.
+Creating an alternative layout goes beyond [[adding or removing features|Customising TiddlyWiki's user interface]] from the default interface, which also known as [[standard layout|$:/core/ui/PageTemplate]], and allows you to create an entirely new layout from scratch.
```

Comment:
Is this a proper sentence? I think if you would use 3 simple sentences it would be easier to read.